qiskit/converters/ast_to_dag.py

```diff
@@ -26,36 +26,36 @@
-from qiskit.extensions.standard.h import HGate
-from qiskit.extensions.standard.i import IGate
-from qiskit.extensions.standard.s import SGate
-from qiskit.extensions.standard.s import SdgGate
-from qiskit.extensions.standard.t import TGate
-from qiskit.extensions.standard.t import TdgGate
-from qiskit.extensions.standard.u1 import U1Gate
-from qiskit.extensions.standard.u2 import U2Gate
-from qiskit.extensions.standard.u3 import U3Gate
-from qiskit.extensions.standard.x import XGate
-from qiskit.extensions.standard.y import YGate
-from qiskit.extensions.standard.z import ZGate
-from qiskit.extensions.standard.rx import RXGate
-from qiskit.extensions.standard.ry import RYGate
-from qiskit.extensions.standard.rz import RZGate
-from qiskit.extensions.standard.rxx import RXXGate
-from qiskit.extensions.standard.rzz import RZZGate
-from qiskit.extensions.standard.u1 import CU1Gate
-from qiskit.extensions.standard.u3 import CU3Gate
-from qiskit.extensions.standard.h import CHGate
-from qiskit.extensions.standard.rx import CRXGate
-from qiskit.extensions.standard.ry import CRYGate
-from qiskit.extensions.standard.rz import CRZGate
+from qiskit.circuit.barrier import Barrier
+from qiskit.circuit.library.standard_gates.x import CCXGate
+from qiskit.circuit.library.standard_gates.swap import CSwapGate
+from qiskit.circuit.library.standard_gates.x import CXGate
+from qiskit.circuit.library.standard_gates.y import CYGate
+from qiskit.circuit.library.standard_gates.z import CZGate
+from qiskit.circuit.library.standard_gates.swap import SwapGate
+from qiskit.circuit.library.standard_gates.h import HGate
+from qiskit.circuit.library.standard_gates.i import IGate
+from qiskit.circuit.library.standard_gates.s import SGate
+from qiskit.circuit.library.standard_gates.s import SdgGate
+from qiskit.circuit.library.standard_gates.t import TGate
+from qiskit.circuit.library.standard_gates.t import TdgGate
+from qiskit.circuit.library.standard_gates.u1 import U1Gate
+from qiskit.circuit.library.standard_gates.u2 import U2Gate
+from qiskit.circuit.library.standard_gates.u3 import U3Gate
+from qiskit.circuit.library.standard_gates.x import XGate
+from qiskit.circuit.library.standard_gates.y import YGate
+from qiskit.circuit.library.standard_gates.z import ZGate
+from qiskit.circuit.library.standard_gates.rx import RXGate
+from qiskit.circuit.library.standard_gates.ry import RYGate
+from qiskit.circuit.library.standard_gates.rz import RZGate
+from qiskit.circuit.library.standard_gates.rxx import RXXGate
+from qiskit.circuit.library.standard_gates.rzz import RZZGate
+from qiskit.circuit.library.standard_gates.u1 import CU1Gate
+from qiskit.circuit.library.standard_gates.u3 import CU3Gate
+from qiskit.circuit.library.standard_gates.h import CHGate
+from qiskit.circuit.library.standard_gates.rx import CRXGate
+from qiskit.circuit.library.standard_gates.ry import CRYGate
+from qiskit.circuit.library.standard_gates.rz import CRZGate
```

Comment:
are we missing some gate CX4 etc

Reply:
have to cross check for all gates. I think a new `stdgate.inc` library is needed for qasm as it's different than the original `qelib1.inc`.